src/Microsoft.OpenApi/Models/OpenApiComponents.cs

```diff
@@ -293,8 +293,10 @@ public void SerializeAsV3(IOpenApiWriter writer)
                     }
                 });
             
-            // pathItems
-            writer.WriteOptionalMap(
+            // pathItems - only present in v3.1
+            if(version == OpenApiSpecVersion.OpenApi3_1)
```

Comment:
should it be >= instead? or should we introduce a "futureMajorVersion" enum member to facilitate upgrading the code when 4.0 comes around?

Reply:
Will moonwalk(4.0) have all the additional elements in 3.1 e.g. webhooks?

Reply:
I think there are some breaking improvements around enums and other aspects like that.